sdk/mobile-center-distribute/src/main/java/com/microsoft/azure/mobile/distribute/Distribute.java

```diff
@@ -641,21 +649,28 @@ else if (mUnknownSourcesDialog != null) {
                 return;
             }
 
-            /* Check if we have previous stored the update token. */
+            /*
+             * Check if we have previously stored the redirection parameters,
+             * note that distribution group is missing in previous SDK releases.
```

Comment:
I agree with your last sentence but what I concerned was that is just saying "previous". Please track it and replace to a specific version after release.